tools/custom-checks/check-package-json-sort.ts

```diff
@@ -0,0 +1,122 @@
+import fs from "fs-extra";
+import path from "path";
+import { fileURLToPath } from "url";
+import { globby, Options as GlobbyOptions } from "globby";
+import { RuntimeError } from "run-time-error";
+import { isStdLibRecord } from "./is-std-lib-record";
+import { sortPackageJson } from "sort-package-json";
+
+export interface ICheckPackageJsonSort {
+  readonly argv: string[];
+  readonly env: NodeJS.ProcessEnv;
+  /**
+   * The version that will be used as the correct one when checking the sibling
+   * package dependency version declarations.
+   * If you omit this (optional) parameter then the root package.json file
+   * will be parsed to obtain its value at runtime.
+   */
+  readonly version?: string;
```

Comment:
@Leeyoungone Looks from the code that you don't need this because it gets parsed but then the value of it does not get used in the end.

It's okay to leave the `argv` and `env` parameters in because those could be needed later on if we want to make some customizations in how to check is made (for example an environment variable that can turn this specific check off or something - not needed I'm just saying that it's a possibility in the future)